test/test_helper.exs

```diff
@@ -1,14 +1,24 @@
-alias Popcorn.Support.AtomVM
+alias Popcorn.Support.{AtomVM, Browser}
 
-# FIXME: Workaround for the `Popcorn.Support.AtomVM` caching not considering changing the patches
-File.rm_rf!("tmp")
+Path.wildcard("tmp/*") |> List.delete("tmp/modules") |> Enum.each(&File.rm_rf!/1)
+
+target = System.get_env("TARGET", "UNIX") |> String.downcase() |> String.to_atom()
+AtomVM.test_target(target)
 
 for type <- [:eval_elixir, :eval_erlang_module, :eval_erlang_expr] do
   type
   |> AtomVM.ast_fragment()
   |> AtomVM.compile_quoted()
 end
 
-ci_opts = if System.get_env("CI") == "true", do: [max_cases: 1], else: []
+case target do
+  :unix ->
+    Popcorn.ingredients(target: :unix, out_dir: "test/fixtures/unix")
 
-ExUnit.start([capture_log: true, exclude: :long_running] ++ ci_opts)
+  :wasm ->
+    Popcorn.ingredients(target: :wasm, out_dir: "test/fixtures/wasm/static/wasm")
+    Browser.launch()
+end
+
+ci_opts = if System.get_env("CI") == "true", do: [max_cases: 1], else: []
+ExUnit.start([capture_log: true, exclude: [:long_running, skip_target: target]] ++ ci_opts)
```

Comment:
Why CI needs `max_cases: 1`? Also we should consider doing something about long-running tests, excluding them by default ensures nobody will ever run them.